FancyScrollView.podspec

```diff
@@ -0,0 +1,16 @@
+Pod::Spec.new do |s|
+    s.name                  = 'FancyScrollView'
+    s.version               = '0.1.3'
+    s.summary               = 'A SwiftUI ScrollView Designed to imitate the App Store and Apple Music ScrollViews'
+    s.description           = <<-DESC
+    A SwiftUI ScrollView Designed to imitate the App Store and Apple Music ScrollViews (with or without a Parallax Header)
+    DESC
+    s.homepage              = 'https://github.com/nerdsupremacist/FancyScrollView'
+    s.license               = { :type => 'Copyright', :file => 'LICENSE' }
+    s.author                = { 'Mathias Quintero' => 'me@quintero.io' }
+    s.source                = { :git => 'https://github.com/nerdsupremacist/FancyScrollView.git', :tag => "#{s.version}" }
```

Comment:
I've pointed everything to the original. Is this how it should be or should we be pointing to our fork?

Reply:
@igorcferreira Got it! Updated the source to our fork 👍🏼